tardis/io/tests/test_atomic.py

```diff
@@ -48,6 +48,7 @@ def test_atom_data_levels(levels):
 
 
 def test_atom_data_lines(lines):
+    lines = lines.sort_index()
```

Comment:
I think we don't want to overwrite or change pytest fixtures like this. We ran into problems awhile back with tests passing when they were run in a specific order, but not otherwise because of changing the fixtures in tests. 

Probably just save the sorted lines to a new variable?

Reply:
updated